src/arc_utilities/conditional_try.py

```diff
@@ -0,0 +1,13 @@
+import traceback
+
+
+def conditional_try(conditional, function, **kwargs):
```

Comment:
add a `value_on_exception=None` param, and `return value_on_exception` at the end of the `except` catch.

The path of "if condition and function raises" does not return a value, while the other flow paths do return. This is a source for unexpected errors. Adding this param both makes `conditional_try` more versatile, and the behavior on exception more clear.